src/ddbToS3/utility.ts

```diff
@@ -0,0 +1,86 @@
+import _ from 'lodash';
+
+const TTL_FIELD_NAME = '_ttlInSeconds';
+
+/**
+ * parses string type archive config and returns it in Map format.
+ */
+function parseArchiveConfig(archiveConfig: string | undefined): Map<string, number> {
+    const result = new Map<string, number>();
+
+    archiveConfig?.split('|').forEach((archiveEntry) => {
+        if (archiveEntry.trim().length > 0) {
+            const resourceToTTL = archiveEntry.split(',');
+            if (resourceToTTL.length === 2 && resourceToTTL[0].trim() && resourceToTTL[1].trim()) {
+                try {
+                    const ttl = parseInt(resourceToTTL[1], 10);
+                    if (ttl <= 0) {
+                        throw new Error(`TTL time should be greater than 0: ${ttl}`);
+                    }
+                    result.set(resourceToTTL[0].trim(), ttl);
+                } catch (error) {
+                    throw new Error(`Invalid TTL time: ${resourceToTTL[1]}`);
+                }
+            } else {
+                throw new Error(`Invalid archive config: ${archiveEntry}`);
+            }
+        }
+    });
+
+    return result;
+}
+
+/**
+ * returns a new array that only contains records that were removed from TTL elapse
+ */
+function filterRemovedRecordsFromTTL(records: any[]): any[] {
+    return records.filter(
+        (record) =>
+            record.eventName === 'REMOVE' &&
+            record.userIdentity &&
+            record.userIdentity.type === 'Service' &&
+            record.userIdentity.principalId === 'dynamodb.amazonaws.com',
```

Comment:
The remove record doesn't contain the _ttlInSeconds value?

Reply:
I'm curious why you ask that ?